fast64_internal/oot/oot_operators.py

```diff
@@ -82,21 +86,28 @@ def execute(self, context):
 		entranceObj = context.view_layer.objects.active
 		entranceObj.ootEmptyType = "Entrance"
 		entranceObj.name = "Entrance"
-		entranceObj.ootEntranceProperty.actor.actorParam = "0x0FFF"
+		entranceObj.fast64.oot.actor.actorID = 'ACTOR_PLAYER'
+		entranceObj.fast64.oot.actor.actorKey = '0000'
+		setattr(entranceObj.fast64.oot.actor, '0000.type', '0F00')
+		setattr(entranceObj.fast64.oot.actor, '0000.props1', '0xFF')
```

Comment:
I will say I'm very much not a fan of hex without the proper 0x prefix
I also saw hex without 0x in the UI but it may be up to the xml

Reply:
for the actor keys it's this way to keep an easy compatibility between fast64 versions because of decomp still evolving (avoids having a headache everytime fast64 gets broken because of decomp changes)

I think I can add the 0x prefix but would it be odd to have something like: ``fast64.oot.actor.0x0000.type``?